services/src/main/java/org/keycloak/services/resources/WelcomeResource.java

```diff
@@ -255,7 +255,7 @@ private Theme getTheme() {
     }
 
     protected String getAdminCreationMessage() {
```

Comment:
Does this actually get used anywhere? Perhaps we can get rid of this code.

Reply:
The [`adminUserCreationMessage` property](https://github.com/keycloak/keycloak/blob/0eb0281bf222d7781dbf2bdc084a4f73f06c8bc7/services/src/main/java/org/keycloak/services/resources/WelcomeResource.java#L220) doesn't seem to be used, you're right. I don't think we need to wait to 27 before removing it.

Reply:
Actually, we can't remove as custom themes might be using it. So we need to wait to 27...

Reply:
Should I capture this as a follow up issue then?

Reply:
Ok, then let's log a task to remove it in the next major